bots/mlbridge/src/main/java/org/openjdk/skara/bots/mlbridge/ArchiveWorkItem.java

```diff
@@ -118,21 +119,41 @@ private Repository materializeArchive(Path scratchPath) {
         }
     }
 
-    private boolean ignoreComment(HostUser author, String body, ZonedDateTime createdTime, ZonedDateTime lastDraftTime) {
+    static String filterOutCommands(String body) {
+        var filteredBody = new StringBuilder();
+        boolean readingMultiLineCommandArgs = false;
+        for (var line : body.split("\\R")) {
+            var commandMatcher = EXECUTION_COMMAND_PATTERN.getPattern().matcher(line);
+            if (commandMatcher.matches()) {
+                readingMultiLineCommandArgs = false;
+                var command = commandMatcher.group(1).toLowerCase();
+                if (Arrays.stream(CommandNameEnum.values()).anyMatch(commandNameEnum -> commandNameEnum.name().equals(command))
+                        && CommandNameEnum.valueOf(command).isMultiLine()) {
+                    readingMultiLineCommandArgs = true;
+                }
+            } else {
+                if (!readingMultiLineCommandArgs) {
+                    filteredBody.append(line).append(System.lineSeparator());
+                }
+            }
+        }
+        return filteredBody.toString().strip();
+    }
+
+    private boolean ignoreComment(HostUser author, String body, ZonedDateTime createdTime, ZonedDateTime lastDraftTime, boolean isComment) {
```

Comment:
Introduced argument `isComment` to this method because I think in any case, no review should be ignored. But according to the previous logic, if a review whose body only contains command, the review would be ignored.

Reply:
I'm not sure about this. We already send email when someone approves a PR. I don't think we need to send emails with an empty review.

Reply:
According to the previous logic, for example, if we approve a PR and the review body only contains command like "/reviewers 2", then this review will be ignored. This issue is what I want to resolve.

But now I also think something in my logic is wrong.

Reply:
See this PR that I reviewed this morning: https://github.com/openjdk/jdk/pull/14261

I didn't put any text in the review comment/body, but we still had this email sent: https://mail.openjdk.org/pipermail/build-dev/2023-June/039655.html

Reply:
Yes, empty body is good. But if and only if the review body contains command, it will be ignored. 

Just like when you are approving the pr, you put "/reviewers 2"(or any other command) in the review body.

Reply:
Ah, that's pretty bad, but I see what you mean now after having studied the code.